tests/src/Context/ConfigurationContext.php

```diff
@@ -165,4 +151,12 @@ private function clearCache(): void
 
         $application->run($input);
     }
+
+    /**
+     * @Given I rebuild Webpack Encore assets
+     */
+    public function rebuildYarn(): void
+    {
+        shell_exec("yarn encore dev");
```

Comment:
Idea: is it possible to get Symfony env here (with `$APP_ENV` or something similar)?

Reply:
It should be possible to use `ezplatform:encore:compile` here (from https://github.com/ezsystems/ezplatform/blob/master/composer.json#L110)

It will rely on APP_ENV (https://github.com/ezsystems/ezplatform-core/pull/7#issuecomment-484425523)